web/src/click_handlers.js

```diff
@@ -1,7 +1,6 @@
 import $ from "jquery";
 import assert from "minimalistic-assert";
-import tippy from "tippy.js";
-
+import tippy, {createSingleton} from "tippy.js";
```

Comment:
It's probably better to leave that open line of whitespace above the comment below.